pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -308,7 +307,7 @@ effectiveStdenv.mkDerivation {
     glib
     glog
     pcre2
-    protobuf_29
+    protobuf_21
```

Comment:
I am on mobile right now so I can't check but was that downgrade here from the beginning? I would rather like up update if am honest 😅😅😅

Reply:
Ah yes I forgot to put a comment here: it's because of this: https://github.com/opencv/opencv/blob/b5f5540e8ab5f94afc43e30b77f42afb90635fe4/cmake/OpenCVFindProtobuf.cmake#L81C4-L81C17

basically, this errors happened when I try to build opencv : protobuf cannot be greater than 22 if `CMAKE_CXX_STANDARD` is less than 17. I assumed that nixpkgs' stdenv was in that case, so I downgraded protobuf. Maybe there's a better way?

Reply:
Can you add that as a comment? Bumping cxx standard usually causes lots of regressions.